src/registry/number/edit.tsx

```diff
@@ -0,0 +1,221 @@
+import {NumberComponentSchema} from '@open-formulieren/types';
+import {useFormikContext} from 'formik';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {
+  BuilderTabs,
+  ClearOnHide,
+  Description,
+  Hidden,
+  IsSensitiveData,
+  Key,
+  Label,
+  PresentationConfig,
+  Registration,
+  SimpleConditional,
+  Suffix,
+  Tooltip,
+  Translations,
+  Validate,
+  useDeriveComponentKey,
+} from '@/components/builder';
+import {Checkbox, NumberField, TabList, TabPanel, Tabs} from '@/components/formio';
+import {getErrorNames} from '@/utils/errors';
+
+import {EditFormDefinition} from '../types';
+
+/**
+ * Form to configure a Formio 'number' type component.
+ */
+const EditForm: EditFormDefinition<NumberComponentSchema> = () => {
+  const [isKeyManuallySetRef, generatedKey] = useDeriveComponentKey();
+  const {errors} = useFormikContext<NumberComponentSchema>();
+
+  const erroredFields = Object.keys(errors).length
+    ? getErrorNames<NumberComponentSchema>(errors)
```

Comment:
How should I read this? It's a function from FormikErrors -> string[]  ... But what do the angle brackets tell me?

Reply:
`useFormikContext<Values>` type var describes the shape/type of `values` (and therefore also the structure of `errors`, but there the type of every value in the key-value pair is `string`)

`getErrorNames` is our own function, not from formik, but is inspired by it and follows the same approach. I think the `getErrorNames` type arg might be possible to remove because it can be inferred from the `error` argument type :thinking:

Reply:
Some quick testing - it is indeed not needed, I will finish the date component PR and then do a refactor round!

![image](https://github.com/open-formulieren/formio-builder/assets/5518550/465b07ee-6a40-49e2-9b3d-f417407becb4)